content/en/blogs/Will Cloud Native WebAssembly Replace Docker_.md

```diff
@@ -0,0 +1,90 @@
+# ****Will Cloud Native WebAssembly Replace Docker?****
+
+Holding a brand new form, WebAssembly excels in its portability, small size, fast loading and compatibility. Given its good security, compatibility, high efficiency and lightweight, WebAssembly is an ideal technology for applications based on sandbox schemes and has got noticed from communities of containers, function computing, IoT and edge computing. What is WebAssembly? Is it capable of replacing Docker?
+
+This article is based on the share by Second State CEO Michael Yuan in 2020 KubeSphere Meetup.
+
+## ***Background***
```

Comment:
Same here.

Reply:
Please take a look at this style guide: https://github.com/kubesphere/website/blob/master/KubeSphere%20Documentation%20Style%20Guide.md